modules/virt-configuring-vm-dpdk.adoc

```diff
@@ -75,7 +75,7 @@ spec:
 $ oc apply -f <file_name>.yaml
 ----
 
-. Configure the guest operating system. The following example shows the configuration steps for {op-system-base} 8 OS:
+. Configure the guest operating system. The following example shows the configuration steps for {op-system-base} 9 OS:
```

Comment:
Although it's not part of your direct changes, "OS" should not be used: 

![Screenshot from 2024-10-30 14-18-43](https://github.com/user-attachments/assets/afd763b8-6fdd-4750-a35f-78e1b938dcb1)

Reply:
https://redhat-documentation.github.io/supplementary-style-guide/#operating-system

Reply:
Done